scss/civicrm/contact/pages/_events.scss

```diff
@@ -279,6 +279,10 @@
   }
 
   .custom-group {
+    .form-layout-compressed {
+      margin-bottom: -10px;
```

Comment:
magic number, why?

Reply:
This one is to uplift tables in custom groups like below

![image](https://user-images.githubusercontent.com/26058635/37822122-9a280e70-2eab-11e8-963d-4dc0cda0e481.png)